python/composio/client/collections.py

```diff
@@ -516,7 +516,7 @@ def _handle_callback(
         try:
             return callback(data)
         except BaseException:
-            self.logger.info(
+            self.logger.error(
```

Comment:
Changing the log level from `info` to `error` in the `_handle_callback` method is appropriate for handling exceptions. This ensures that errors are more visible in the logs, which is crucial for debugging.

Reply:
thanks